mage_integrations/mage_integrations/destinations/gcs/templates/config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "bucket": null,
+  "file_type": "parquet",
+  "object_key_path": null,
+  "table": null,
+  "date_partition_format": null,
+  "project": "your-gcp-project-id"
```

Comment:
I think you need `service_account_json` here.

Reply:
In this case, I assumed [Workload identity will solve it](https://cloud.google.com/kubernetes-engine/docs/how-to/hardening-your-cluster#use_least_privilege_sa), but they may be cases where users won't be using workload identity and may be best to use service_account_json or should we make service_account_json optional?

Reply:
I see. Right not, the client is initiated with the `service_account_json`:
```
    def __init__(self, **kwargs):
        """
        Initialize a new GoogleCloudStorage object.
        Args:
            **kwargs: Additional arguments to pass to the superclass constructor.
        """        
        super().__init__(**kwargs)
        self.client = storage.Client.from_service_account_json(self.config['service_account_json'])
```
If we make `service_account_json` optional, let's make sure the code handles that.